Update nominations_reminder_3_days_left.html.erb

katjonzz opened the pull request:
Hi Chris - this will only go out to people who have nomination rights, right?

I'm not sure how to tell what the subject line will be for this email. I don't see it in this code. Can we make sure it doesn't mention Retro Hugos and that the year is right?

And just want to confirm the variables are set up for this year's Hugos:
@hugo_nom_deadline = 11:59 pm PDT, 15 March 2022 
@member_login_url = https://registration.chicon.org/
@hugo_ballot_pub_month = April
@worldcon_public_name = Chicon 8
@start_day_informal = September 1
@end_day_informal = September 5
@worldcon_year = 2022
@worldcon_year_before = 2021
@email_hugo_help = hugo-help@chicon.org
@organizers_names_for_signature = Kat Jones and Nicholas Whyte

Thank you!

katjonzz committed:
Update nominations_reminder_3_days_left.html.erb

Hi Chris - this will only go out to people who have nomination rights, right?

I'm not sure how to tell what the subject line will be for this email. I don't see it in this code. Can we make sure it doesn't mention Retro Hugos and that the year is right?

And just want to confirm the variables are set up for this year's Hugos:
@hugo_nom_deadline = 11:59 pm PDT, 15 March 2022 
@member_login_url = https://registration.chicon.org/
@hugo_ballot_pub_month = April
@worldcon_public_name = Chicon 8
@start_day_informal = September 1
@end_day_informal = September 5
@worldcon_year = 2022
@worldcon_year_before = 2021
@email_hugo_help = hugo-help@chicon.org
@organizers_names_for_signature = Kat Jones and Nicholas Whyte

offbyone commented on app/views/nomination_mailer/nominations_reminder_3_days_left.html.erb:
```diff
@@ -50,38 +52,34 @@
-  <a href=<%= @wsfs_constitution_link %>>here</a>.
+  A printable version of the WSFS Constitution and Hugo Award Nominating ballot 
+  are available online 
+  <a href=<https://chicon.org/home/whats-happening/hugo-awards/>>here</a>, but
```
I'd rather you left this with `<%= @wsfs_constitution_link %>` instead.

katjonzz replied:
That's fine as long as @wsfs_constitution_link can be set to https://chicon.org/home/whats-happening/hugo-awards/ since that's where both the printable ballot and WSFS constitution links live.

offbyone replied:
Easy.

katjonzz committed:
Update nominations_reminder_3_days_left.html.erb

That's fine as long as @wsfs_constitution_link can be set to https://chicon.org/home/whats-happening/hugo-awards/ since that's where the links to the printable ballot and the constitution both live. Thanks!
